cmake/ia2.cmake

```diff
@@ -309,12 +320,15 @@ function(add_ia2_call_gates NAME)
   )
 
   add_custom_target(${NAME}-rewrite
-    DEPENDS ${CALL_GATE_SRC} ${CALL_GATE_HDR} ${LD_ARGS_FILES} ${REWRITTEN_SOURCES})
+    DEPENDS ${CALL_GATE_SRC} ${CALL_GATE_HDR}
+            ${LD_ARGS_FILES} ${OBJCOPY_ARGS_FILES} ${REWRITTEN_SOURCES})
 
   add_library(${CALL_GATE_TARGET} SHARED ${CALL_GATE_SRC})
   add_dependencies(${CALL_GATE_TARGET} ${NAME}-rewrite)
+  # This is needed to enable LIBIA2_DEBUG=1
+  target_compile_definitions(${CALL_GATE_TARGET} PRIVATE IA2_ENABLE=1)
   if(LIBIA2_DEBUG)
-    target_compile_definitions(${CALL_GATE_TARGET} PRIVATE LIBIA2_DEBUG=1)
+    target_compile_definitions(${CALL_GATE_TARGET} PRIVATE LIBIA2_DEBUG=1 LIBIA2_ENABLE=1)
```

Comment:
I don't see any other mentions of `LIBIA2_ENABLE` (with "LIB") in the codebase--is this actually needed?

Reply:
ah I forgot to commit the typo fix, it should be IA2_ENABLE